packages/edit-site/src/components/dataviews/in-filter.js

```diff
@@ -2,62 +2,100 @@
  * WordPress dependencies
  */
 import {
-	__experimentalInputControlPrefixWrapper as InputControlPrefixWrapper,
-	SelectControl,
+	Button,
+	privateApis as componentsPrivateApis,
+	Icon,
 } from '@wordpress/components';
+import { check, chevronDown } from '@wordpress/icons';
 import { __, sprintf } from '@wordpress/i18n';
 
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
 export const OPERATOR_IN = 'in';
 
+const { DropdownMenuV2, DropdownMenuItemV2, DropdownMenuSeparatorV2 } = unlock(
+	componentsPrivateApis
+);
```

Comment:
Could you try to use the `DrodpdownMenuV2Ariakit` component? The `DropdownMenuV2` component is likely going to removed soon.

Reply:
I tried, and left some comments at https://github.com/WordPress/gutenberg/pull/56110#discussion_r1399224211